src/Cache/Errors/AlreadyExistsError.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace Momento\Cache\Errors;
+
+class AlreadyExistsError extends MomentoServiceError {}
```

Comment:
nit - does PHP restrict to a single class per file?  Just curious if we could consolidate the error classes.

Reply:
Haha, turns out it doesn't despite my initial confusion about the PHP autoloader. I figured out how to consolidate these but didn't want to delay delivering this to fix that. Consolidating these is definitely part of my planned cleanup though.